src/pages/basic_data/app_admin/central_import/CentralImportDuplicationCheckPage.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Divider, Link as MuiLink, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFetch } from '../../../../utils/hooks/useFetch';
+import { Registration } from '../../../../types/registration.types';
+import { PublicationsApiPath } from '../../../../api/apiPaths';
+import { useParams } from 'react-router-dom';
+import { SyledPageContent } from '../../../../components/styled/Wrappers';
+import { PageSpinner } from '../../../../components/PageSpinner';
+import NotFound from '../../../errorpages/NotFound';
+import { useEffect } from 'react';
+import { stringIncludesMathJax, typesetMathJax } from '../../../../utils/mathJaxHelpers';
+import { CentralImportDuplicateSearch } from './CentralImportDuplicateSearch';
+
+export const CentralImportDuplicationCheckPage = () => {
+  const { t } = useTranslation('basicData');
+  const { identifier } = useParams<{ identifier: string }>();
```

Comment:
fyi: bare så du vet det så gikk vi på ett punkt over til å prøve å bruke hele id i stedet for bare identifier i URL. eks: https://frontend.dev.nva.aws.unit.no/basic-data/institutions?id=https%3A%2F%2Fapi.dev.nva.aws.unit.no%2Fcustomer%2Ffe263b70-b5e9-40de-8566-3a69afb29c5d

problemet er at det ser helt forjævlig ut da. Bruker fortsatt bare identifier for registreringer også, så tenker det er greit å bruke det her også :)

Reply:
trenger ikke endre altså